src/www/ui/core-schema.dat

```diff
@@ -1901,8 +1970,8 @@
   $Schema["SEQUENCE"]["license_map_license_map_pk_seq"]["CREATE"] = "CREATE SEQUENCE \"license_map_license_map_pk_seq\"";
   $Schema["SEQUENCE"]["license_map_license_map_pk_seq"]["UPDATE"] = "SELECT setval('license_map_license_map_pk_seq',(SELECT greatest(1,max(license_map_pk)) val FROM license_map))";
 
-  $Schema["SEQUENCE"]["copyright_decision_pk_seq"]["CREATE"] = "CREATE SEQUENCE \"copyright_decision_pk_seq\"";
-  $Schema["SEQUENCE"]["copyright_decision_pk_seq"]["UPDATE"] = "SELECT setval('copyright_decision_pk_seq',(SELECT greatest(1,max(copyright_decision_pk)) val FROM copyright_decision))";
```

Comment:
Please revert changes (removal of sequence).

https://travis-ci.org/fossology/fossology/jobs/597459220#L3166

Reply:
Updated changes in latest commit

Reply:
I am sorry, which commit you are referring to as I can not see any new commit since the comment?

Reply:
I have pushed my changes to feat/spasht_5.0

Reply:
But you'll need to create a PR from `feat/spasht_5.0` to `feat/newagent/spasht`.

Reply:
creating an new per, which is not directed to master, might not be the best idea and we also only want to have merges on master and not on branches. Better update this PR directly with a push to the corresponding branch